### DISCUSSION TITLE
Need help with application

### DISCUSSION THREAD

**Discussion by EboLiK1** [community/community#43998] [GitHub Education]
Hey everyone, as you can see from the title, I applied for the GitHub Student Development Pack but ran into a problem... I'm not getting approved because I'm far away from the college when I'm actually I live very close. I wrote to support, but there is no answer for 3 days.

P.S. Sorry for my english, I'm writing through a translator)

**Comment by samus-aran** [comment]
Hi @EboLiK1 

We're happy you're here 👋 

Ensure you have enabled browser location so the application can accurately discover near by schools and campuses. For remote student applicants: please ensure that your evidence indicates that you are a remote student. If possible, we encourage students to apply whilst on campus. This will assist us in confirming your student status.

Let us know how you get on. 👍

**Reply by EboLiK1** [reply]
Hi, I allowed to determine my location, but it does not work. I also tried to connect to the Internet of college (in the usual way and through the cable). I did it through the cable once, but the next time a mistake appeared again
